lang/test/org/partiql/lang/syntax/SqlParserTestBase.kt

```diff
@@ -196,12 +196,12 @@ abstract class SqlParserTestBase : TestBase() {
      *
      * Verify that the result matches the original without metas.
      */
-    private fun pigExprNodeTransformAsserts(actualExprNode: ExprNode) {
+    private fun roundTripPigAstToExprNode(actualExprNode: ExprNode) {
```

Comment:
These are still `assertions`, right? I would rename this to `assertRoundTripPigAstToExprNode`.

Reply:
Done in the next revision.